client/me/profile/style.scss

```diff
@@ -42,7 +42,7 @@
 }
 
 .profile__submit-button-wrapper {
-	margin-bottom: 0;
+	margin-bottom: 24px;
```

Comment:
Hm, if we remove this style at all - I don't see any difference, since we have `    margin-bottom: 1.5em;` for `p`.
Did I miss something?

Reply:
Thanks for spotting this! It should have been kept as `margin-bottom: 0`, I updated the padding earlier in the iteration where we added a Developer section instead of a page. This is no longer needed. I'll revert the change.